fabric/src/main/java/com/wynntils/fabric/mixins/RenderTargetMixin.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.fabric.mixins;
+
+import com.mojang.blaze3d.pipeline.RenderTarget;
+import com.wynntils.gui.render.RenderUtils;
+import org.lwjgl.opengl.GL30;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.ModifyArgs;
+import org.spongepowered.asm.mixin.injection.Slice;
+import org.spongepowered.asm.mixin.injection.invoke.arg.Args;
+
+/*
+ * See https://gist.github.com/burgerguy/8233170683ad93eea6aa27ee02a5c4d1
+ */
+@Mixin(RenderTarget.class)
+public class RenderTargetMixin {
+    @ModifyArgs(
+            method = "createBuffers",
+            at =
+                    @At(
+                            value = "INVOKE",
+                            target =
+                                    "Lcom/mojang/blaze3d/platform/GlStateManager;_texImage2D(IIIIIIIILjava/nio/IntBuffer;)V",
+                            ordinal = 0,
+                            remap = false))
+    public void init(Args args) {
+        if (RenderUtils.isStencilEnabled()) {
```

Comment:
I'm not really fond of this. I fear it has a high risk of conflict with other mods. Is there some other way you can go about to achieve what you need? (This goes for both `@ModifyArgs`)

Reply:
I doubt there will be much conflict, as I feel any other modification would be also for enabling stencil.

Reply:
We agreed to merge this on the basis that if this creates problems, we revert it